src/Frontend/components/portal/signup/Form.js

```diff
@@ -107,12 +110,33 @@ export default class Form extends Component {
           helper={this.helper}
           value={this.helper.getValue(this.state.data.locale, 'locale')}
           options={{ de: 'Deutsch (Deutschland)', en: 'English (USA)' }} />
-        <LoadableButtonBar btnLabel={this.helper.getFormFieldAlias('button')} progress={this.state.progress} />
+          <Recaptcha
+              sitekey={siteKey}
+              render='explicit'
+              onloadCallback={callback}
+              verifyCallback={this.verifyCallback.bind(this)}
+          />
+          <LoadableButtonBar btnLabel={this.helper.getFormFieldAlias('button')} progress={this.state.progress} />
```

Comment:
well, actually the react eslint plugin isn't configured properly, need to fix this when I've got time to work on Sententiaregum again :)